application/controllers/Files.php

```diff
@@ -0,0 +1,18 @@
+<?php
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Files extends MY_Controller {
+
+	function __construct()
+	{
+		parent::__construct();
+	}
+
+	public function index()
+	{
+		$data = [];
```

Comment:
Ik denk dat het toch handiger is om `$data = [];` weg te halen. Ik heb het nog even nagezocht en het is overbodig om `$data` te defineren in CodeIgniter.